tests/selenium/framework/page-objects/SignUpPage.js

```diff
@@ -75,6 +81,12 @@ class SignUpPage extends BasePage {
     this.waitForPresence(pageLoadElement);
   }
 
+  navigateToOie() {
+    const pageLoadElement = SignUpPage.getPageLoadElement();
```

Comment:
can we not call getPageLoadElement directly?

Reply:
requires refactoring of the whole class. will keep this for future work